recipes/dessert/cheesecake.md

```diff
@@ -0,0 +1,24 @@
+# Cheesecake
+
+Adapted from [Chef John's cheesecake](http://foodwishes.blogspot.com/2010/10/new-york-style-sunshine-cheesecake.html). Makes 4 ramekins + glass tupperware.
```

Comment:
> Makes 4 ramekins + glass tupperware

Could you change this to "Makes one small glass tupperware in addition to 4 ramekins"?